compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeWindow.uikit.kt

```diff
@@ -187,10 +193,24 @@ internal actual class ComposeWindow : UIViewController {
         layer.setContent(content = content)
     }
 
+    override fun viewWillTransitionToSize(
+        size: CValue<CGSize>,
+        withTransitionCoordinator: UIViewControllerTransitionCoordinatorProtocol
+    ) {
+        layer.setDensity(density)
+        val scale = layer.layer.contentScale
```

Comment:
contentScale also should return density. Better to not use two sources of truth. Can we rewrite code, so we get density always from `layer.layer.contentScale`?

Reply:
Done